zpeclient.go

```diff
@@ -99,11 +102,23 @@ func (c *Cache) parseData(item *v1.AthenzDomain) (roleMappings, error) {
 			continue
 		}
 		roleName := string(role.Name)
+		// Handle trust domains: if string(role.Trust) != ""
+		// For a trusted domain, role object will only contain following items:
+		// modified, name, trust.
+		// Note: logic may change depend on how signature validation is implemented.
+		if role.Trust != "" {
+			var err error
+			role.RoleMembers, err = c.processTrustDomain(role.Trust, item, roleName)
+			if err != nil {
+				c.log.Printf("Error occurred when processing trust domain. Error: %v", err)
```

Comment:
can we add a continue statement here ? (since we shouldn't further process this role for any members)

Reply:
done